src/sidebar/media-embedder.js

```diff
@@ -100,6 +101,29 @@ var embedGenerators = [
     return null;
   },
 
+/** Match Internet Archive URLs
+ *  The patterns are:
+ *  1. https://archive.org/embed/WHDH_20151121_043400_The_Tonight_Show_Starring_Jimmy_Fallon?start=360&end=420.3
+ *  2. https://archive.org/details/WHDH_20151121_043400_The_Tonight_Show_Starring_Jimmy_Fallon/start/360/end/420.3
+ *  The patterns are invariant, start and stop are always present.
```

Comment:
> The patterns are invariant, start and stop are always present.

Pedantic point - these are _examples_ rather than patterns. A pattern would be something like a URL template or a regex. eg:

```
https://archive.org/embed/{slug}?start={startTime}&end={endTime}
```

> The patterns are invariant, start and stop are always present.

Are they? If I navigate to most videos from the menu at the top of the page I'll end up on a `/details` URL with no `/start` and `/end` path params.